templates/federated_reporting/psql_wrapper.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# run an arbitrary psql command, either -f or -c
+# expectation is that a line will be output with exit_code=<n>
+# this exit code <n> will then be used to exit
+# This design enables cfengine policy to run a commands promise
+# that can be kept, repaired or failed depending on an exit
+# code from psql since psql can't get different codes from scripts.
+
+set -e
+
+EXIT_CODE=$(psql -qtAX "$@" 2>&1 | awk -F= '{print $2}')
```

Comment:
what if something else fails here? like psql command path not correct? Maybe add some more error handling and/or adjust failed_returncodes in the policy.

Reply:
can you use long options? /me is dumb and doenst know exactly what all qtAX does.
what tuples (so no headers), quiet or is that no headers?, A? X does horizontal tables wich is more friendly for line processing. What if the awk fails? Are we sure to exit badly if the query doesnt return what you expect?

Reply:
Yeah, please use long options and split this to multiple lines. Or at least use `pipefail`.

Reply:
Cool. I split it with my latest change.